### PR TITLE
2024-prisoners-dilemma

### PR DESCRIPTION
**Pull request by carinaschrenk**
Article of round 3 on the topic of prisoners dilemma

### CODE REVIEW

**Comment by carinaschrenk** [comment]
# Internal Feedback Round 3B from Google Poll 05.12-08.12 

#### Thoughts 
- Too much text x3
  * The third paragraph is too long
- Hard to follow, confusing
  * Whats the point?
- The topic is interesting 
- Its very nicely explained

#### Ideas 
- Adding labels or clarifying who is cooperating and who is refusing in each scenario in the figure would be good 

#### A general comment:
Just because something is listed in the feedback, does not mean you have to implement it. 
This list is simply a collection of everything commented in the google poll.
I tried to structure the comments and merge similar ones and count how often this point was mentioned.

**Comment by evos96** [comment]
External feedback:

Part of participants that are computer science students/graduates: 71%

Part of participants that would read the booklet if it was not for this survey: 28 %

Clarity of the article:
Very clear: 57 %
Clear: 42 %
Not clear: 0 %

Technical vs. Non-technical:
Good balance between technical depth and readability: 85 %
The article was too technical: 15 %

Engagement:
I stayed engaged throughout: 57 %
I lost interest midway: 42 %

Structure of the article:
The structure is clear and easy to follow: 57 %
The structure was confusing: 42 %

Part of participants that understand the purpose of the article: 71 %

Part of participants that think the drawing helps to understand the meaning of the article:  85 %

Factual errors/inaccuracies or additional comments:
-I think there's an error in the second part where it should be 5 instead of 3 coins (case where the opponent cooperates and you refuse)
